CONTRIBUTING.md

```diff
@@ -199,16 +199,7 @@ If you want to read about using Atom or developing packages in Atom, the [Atom F
 
 #### Local development
 
-All packages can be developed locally, by checking out the corresponding repository and registering the package to Atom with `apm`:
-
-```
-$ git clone url-to-git-repository
-$ cd path-to-package/
-$ apm link -d
-$ atom -d .
-```
-
-By running Atom with the `-d` flag, you signal it to run with development packages installed. `apm link` makes sure that your local repository is loaded by Atom.
+All packages can be developed locally. For instructions on how to do this, see [Contributing to Official Atom Packages][contributing-to-official-atom-packages] in the [Atom Flight Manual](http://flight-manual.atom.io).
```

Comment:
@ungb @rsese We should add documentation for Local development of Atom Core here (Or link to the not yet existing section in the flight-manual)

I opened an issue https://github.com/atom/flight-manual.atom.io/issues/382 ⚡️